boards/PDM/Inc/Io/Io_Aux1Aux2Efuse.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
+
+#include <assert.h>
+#include <math.h>
+#include "App_Efuse.h"
+#include "Io_Efuse.h"
+#include "Io_CurrentSense.h"
+
+/**
+ * Initialize the Aux1Aux2 efuse
+ * @param hspi Handle to the SPI peripheral used for the Efuse
+ */
+void Io_Aux1Aux2Efuse_Init(SPI_HandleTypeDef *hspi);
+
+/**
+ * Enable the Aux 1 channel output
+ */
+void Io_Aux1Aux2Efuse_Aux1Enable(void);
+
+/**
+ * Disable the Aux 1 channel output
```

Comment:
```suggestion
 * Disable the Aux1 channel output for the Aux1Aux2 efuse.
```

Reply:
Can u make sure these are also applied to the functions below.

Reply:
Done